angular.js

```diff
@@ -92,6 +92,8 @@ module.exports = {
         'rxjs/no-compat': 'warn',
         'rxjs/no-exposed-subjects': 'warn',
         'rxjs/no-ignored-observable': 'warn',
+        // fixme: this rule has been turned off as it produced errors
+        'rxjs/no-implicit-any-catch': 'off',
```

Comment:
Should be in `// DISABLED RULES FROM EXTENDED CONFIGS` section above, not in `// ADDITIONAL RULES` :pray: